sylt-macro/src/macro.rs

```diff
@@ -131,3 +131,53 @@ pub fn derive_numbered(item: proc_macro::TokenStream) -> proc_macro::TokenStream
     };
     proc_macro::TokenStream::from(item)
 }
+
+/// Timed macro
+///
+/// Time a function and print the output.
+///
+/// # Examples
+///
+/// ```
+/// #[sylt_macro::timed]
+/// fn hej() {
+///     println!("Hej");
```

Comment:
Maybe not in swedish?
```suggestion
/// fn hi() {
///     println!("Hi");
```